client/src/app/pages/package-list/package-list.tsx

```diff
@@ -204,13 +206,12 @@ export const PackageList: React.FC = () => {
                           {item.decomposedPurl?.type}
                         </Td>
                         <Td
-                          width={10}
-                          modifier="truncate"
-                          {...getTdProps({ columnKey: "path" })}
+                          width={30}
+                          {...getTdProps({ columnKey: "qualifiers" })}
                         >
-                          {item.decomposedPurl?.path}
+                          {item.decomposedPurl?.qualifiers && <PackageQualifiers value={item.decomposedPurl?.qualifiers} />}
```

Comment:
It might be a matter of style but I think we need to be safer here and by doing so try to be consistent by using `condition ? <render value> : null` kind of approach.